persistence/src/main/java/org/mskcc/cmo/metadb/persistence/neo4j/MetadbSampleRepository.java

```diff
@@ -17,23 +17,28 @@
 
 @Repository
 public interface MetadbSampleRepository extends Neo4jRepository<MetadbSample, UUID> {
-    @Query("MATCH (sm: Sample {metaDbSampleId: $metaDbSampleId}) "
-            + "RETURN sm")
+    @Query("MATCH (s: Sample {metaDbSampleId: $metaDbSampleId}) "
+            + "RETURN s")
     MetadbSample findAllSamplesById(@Param("metaDbSampleId") UUID metaDbSampleId);
-    
-    @Query("MATCH (s: SampleAlias {value: $igoId.sampleId, namespace: 'igoId'}) "
-        + "MATCH (s)<-[:IS_ALIAS]-(sm: Sample) "
-        + "RETURN sm")
-    MetadbSample findResearchSampleByIgoId(@Param("igoId") SampleAlias igoId); 
 
-    @Query("MATCH (sm: Sample {metaDbSampleId: $metaDbSampleId})"
-            + "MATCH (sm)<-[:IS_ALIAS]-(s: SampleAlias)"
-            + "RETURN s;")
+    @Query("MATCH (sa: SampleAlias {value: $igoId, namespace: 'igoId'})"
+        + "<-[:IS_ALIAS]-(s: Sample) "
+        + "RETURN s")
+    MetadbSample findResearchSampleByIgoId(@Param("igoId") String igoId);
+
+    @Query("MATCH (sa: SampleAlias {value: $igoId.value, namespace: 'igoId'})"
+        + "<-[:IS_ALIAS]-(s: Sample) "
+        + "RETURN s")
+    MetadbSample findResearchSampleByIgoId(@Param("igoId") SampleAlias igoId);
```

Comment:
Should we rename this to `findResearchSampleBySampleAlias`?

Reply:
good catch